app/views/nic/_base_form.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= selectable_f f, :type, interfaces_types, {},
-                 :class => 'interface_type', :disabled => !f.object.new_record? %>
+                 :class => 'interface_type without_select2', :disabled => !f.object.new_record? %>
```

Comment:
This one doesn't seem to work for me, I find the select2 for interfaces at the top of the edit modal isn't working (i.e. the element doesn't respond to clicks).  Not sure why it wouldn't as it looks identical to the others in this form.